test/framework/components/sprite/component.test.mjs

```diff
@@ -28,16 +38,16 @@ describe("pc.SpriteComponent", function () {
             }
         };
 
-        atlasAsset = new pc.Asset('red-atlas', 'textureatlas', {
-            url: 'base/tests/test-assets/sprite/red-atlas.json'
+        atlasAsset = new Asset('red-atlas', 'textureatlas', {
+            url: 'http://localhost:3000/test/assets/sprites/red-atlas.json'
         });
 
-        spriteAsset = new pc.Asset('red-sprite', 'sprite', {
-            url: 'base/tests/test-assets/sprite/red-sprite.json'
+        spriteAsset = new Asset('red-sprite', 'sprite', {
+            url: 'http://localhost:3000/test/assets/sprites/red-sprite.json'
```

Comment:
any way to have these specified as a relative path?

Reply:
Possibly! I'm not 100% sure. But it did occur to me that I could have a single 'base path' for all assets and import that in each test module. I'm sure this can be improved, but it's for a follow up PR.